resources/views/layout/public.blade.php

```diff
@@ -15,6 +15,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Sora:wght@100..800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://unpkg.com/flickity@2/dist/flickity.min.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.css" rel="stylesheet">
```

Comment:
Do we have any other planned use cases for toast?

Reply:
I added different use cases just in case. About this line - I left it here by accident, we're not using toastr here

Reply:
On which pages?

Reply:
There is a toast blade component with theme options (success, info, warning, error) and with different toast placement. For now it is not used in different places than contact form, but it does not take a lot of space also

Reply:
lets create issue for new design and please add bullet for success page afer form sent